wiki/Reporting_bad_behaviour/id.md

```diff
@@ -13,10 +8,11 @@ Laporan yang dikirim secara tidak sengaja akan diabaikan. Meskipun demikian, par
 
 | Jenis pelanggaran | Pihak yang dapat dihubungi |
 | :-- | :-- |
-| Pesan pribadi yang tidak pantas | **Tidak ada**: harap [abaikan pengguna tersebut](/wiki/Client/Interface/Chat_console#daftar-perintah). Untuk dapat mengabaikan seseorang secara permanen, gunakan fitur daftar pengabaian (*ignore list*) yang ada pada menu [Options](/wiki/Client/Options). |
-| Pelanggaran yang terkait dengan modding dan/atau mapping | NAT atau para anggota [GMT](/wiki/People/The_Team/Global_Moderation_Team) yang memoderasi komunitas mapping/modding, melalui situs web [NAT/BN Management](https://bn.mappersguild.com/reports) |
-| Perilaku yang tidak pantas dari anggota [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) atau [GMT](/wiki/People/The_Team/Global_Moderation_Team) | [Account support team](/wiki/People/The_Team/Account_support_team) melalui alamat email [support@ppy.sh](mailto:support@ppy.sh) |
-| Pelecehan seksual serta segala bentuk kelakuan buruk yang sudah tidak dapat ditoleransi lagi | Harap rujuk artikel [Melaporkan tindak pelecehan](/wiki/Reporting_bad_behaviour/Abuse) |
+| Scamming/Phishing di dalam pesan pribadi | Kirim laporan biasa |
+| Pesan pribadi yang menyinggung/tidak pantas | **Tidak ada**: harap [abaikan pengguna tersebut](/wiki/Client/Interface/Chat_console#daftar-perintah). Untuk dapat mengabaikan seseorang secara permanen, gunakan fitur daftar pengabaian (*ignore list*) yang ada pada menu [Options](/wiki/Client/Options) atau blokir pengguna tersebut melalui tombol di website pada profil mereka |
```

Comment:
```suggestion
| Pesan pribadi yang menyinggung/tidak pantas | **Tidak ada**: harap [abaikan pengguna tersebut](/wiki/Client/Interface/Chat_console#daftar-perintah). Untuk dapat mengabaikan seseorang secara permanen, gunakan fitur daftar pengabaian (*ignore list*) yang ada pada menu [Options](/wiki/Client/Options) atau blokir pengguna tersebut melalui tombol yang ada pada laman profil mereka. |
```

Reply:
there's a full stop at the end of this sentence in the original article fwiw

![image](https://user-images.githubusercontent.com/36564236/166914926-f7cb1520-8aac-4a35-ae85-4ce535fe4f08.png)